pegasus/sites.v3/code.org/public/tools/musiclab.haml

```diff
@@ -0,0 +1,53 @@
+---
+title: Music Lab
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/music-styles.css", rel: "stylesheet"}
+
+%section
+  .wrapper
+    .text-wrapper.col-50
+      %h1 Music Lab
+      %p.body-one
+        =hoc_s("music_lab.tools_page.top.subheading")
+      %p.body-two
+        =hoc_s("music_lab.tools_page.top.desc")
+      %a.link-button{href: CDO.studio_url("/s/music-intro-2024/reset")}
+        =hoc_s("music_lab.button.try_music_lab")
+    %figure.col-45{style: "float: right; margin-top: 1.25rem;"}
+      = view :display_video_thumbnail, id: "music_lab", video_code: "GqlP5s7KTl4", play_button: 'center', letterbox: 'false', download_path: "//videos.code.org/music_lab/Music_Lab_Make_Music_Learn_Code.mp4"
+    .clear
+
+%section.bg-neutral-light
+  .wrapper.centered
+    %h2=hoc_s("music_lab.tools_page.glance.heading")
+    = view :"tools/tools_glance", ages: hoc_s("music_lab.tools_page.glance.ages"), level: hoc_s("music_lab.tools_page.glance.level"), make: hoc_s("music_lab.tools_page.glance.make"), devices: hoc_s("music_lab.tools_page.glance.devices"), browsers: hoc_s("music_lab.tools_page.glance.browsers"), languages: "العربية, Español (España), Español (LATAM), Filipino, Français, हिन्दी, Bahasa Indonesia, Italiano, 日本語, 한국어, فارسی, Polski, Português (Brasil), Português (Portugal), Slovenčina, ภาษาไทย, Türkçe, Tiếng Việt, 繁體字", accessibility: hoc_s("music_lab.tools_page.glance.accessibility")
```

Comment:
I don't believe we support any of the listed accessibility features.  Where did that list come from?